### ISSUE TITLE
Include in source directory not found (Windows), 2nd

### ISSUE THREAD

**Issue by amai2012** [danmar/simplecpp#44]
This looks quite similar to issue #43 :

```
c:\module>dir src\module\extstl

 Verzeichnis von c:\module\src\module\extstl

18.10.2016  17:20    <DIR>          .
18.10.2016  17:20    <DIR>          ..
18.10.2016  17:11                16 B.cpp
18.10.2016  17:12                29 B.h
26.08.2016  13:23             4.150 compilerconfig.h
               3 Datei(en),          4.195 Bytes

c:\module>type src\module\extstl\B.cpp
#include "B.h"

c:\module>type src\module\extstl\B.h
#include "compilerconfig.h"

c:\module>\cppcheck.exe c:\module\src\module\extstl\B.cpp -Ic:\module\src --chec
k-config
Checking c:\module\src\module\extstl\B.cpp ...
[module\module\src\module\extstl\B.h:1]: (information) Include file: "compilerco
nfig.h" not found.
```

**Comment by amai2012** [comment]
At first I specified the cygwin build environment. However a build made with Visual Studio behaves the same way.
It looks related to #43

**Comment by amai2012** [comment]
... and #45 may be the analysis...

**Comment by amai2012** [comment]
Fixed along with #45
